tests/purger/util.py

```diff
@@ -4,9 +4,10 @@
 import os
 from pathlib import Path
 
-from rubin.nublado.purger.models.plan import FileReason
 from safir.pydantic._types import _validate_human_timedelta
 
+from rubin.nublado.purger.models.plan import FileReason
+
 
```

Comment:
Did the import sorter change its mind, or did I manage somehow never to run it over these tests?